### PR TITLE
Feature/add graphql

### PR DESCRIPTION
**Pull request by sudakov-eugene**

### CODE REVIEW

**Comment by sudakov-eugene** [comment]
Do not forget about the rule, @rakvium :hugs: :
`If the maintainer has marked your pull request as invalid, or you have contributed to a repository that isn’t classified with a Hacktoberfest topic, your pull request will not count toward the challenge. Your pull requests will only count toward your participation if they are in a repository with the hacktoberfest topic and once they have been merged, approved by a maintainer or labeled as hacktoberfest-accepted. Additionally, any pull request with the hacktoberfest-accepted label, submitted to any public GitHub repository, with or without the hacktoberfest topic, will be considered valid for Hacktoberfest.`

**Review by rakvium — APPROVED** [review]

File: config/application.rb
Comment (rakvium) [inline config/application.rb:13]: :+1:

**Review by rakvium — APPROVED** [review]

**Pull request merged**
